integrations/pom.xml

```diff
@@ -62,6 +62,8 @@
                     <configuration>
                         <compilerArgs>
                             <arg>-Xlint:all</arg>
+                            <arg>-Xlint:-missing-explicit-ctor</arg>
```

Comment:
Missing explicit constructor is a good warning, we should always have explicit constructors (there are two reasons - javadoc if it should be there, pointing out an error if it should not be there, such as when you forget to add a private constructor for a `Builder`)

Reply:
Removed it.